scripts/generator.py

```diff
@@ -50,7 +50,7 @@ def main():
         oss.fallback(fields)
     cleaner.clean(fields, strict=args.strict)
     finalizer.finalize(fields)
-    fields = subset_filter.filter(fields, args.subset, out_dir)
+    fields = subset_filter.filter(fields, args.subset, args.exclude, out_dir)
```

Comment:
What do you think about moving `exclude` into its own module instead of extending `subset_filter`?

Outside of the functions for loading the files, it looks like the two are independent of each other. Might help when it comes to writing tests to have things more isolated.

Reply:
OK, I'll refactor :+1:

Reply:
I've moved that to `exclude_filter`. Where functions would have been exactly duplicated, I kept these as imports from `subset_filter` to keep things DRY

Reply:
I did a more extensive refactor, see: https://github.com/elastic/ecs/pull/1411#issuecomment-841362269